### ISSUE TITLE
In workflow: relative paths to CommandLineTools prefered

### ISSUE THREAD

**Issue by KerstenBreuer** [rabix/composer#329]
In workflows, I would prefer that the run paths to commandLineTools are given relative to the workflow script instead of absolute. Usually, I keep my workflows next to my CommandLineTools. If relative paths are used, I can move the complete cwl directory without destroying links to CommandLineTools.

For instance, I would like a default behavior like this:
```
steps:
  - id: 01_validate_data
    in:
      - id: feat_in
        source:
          - feat_in
    out:
      - id: validated_feat    
    run: ../tools/01_validate_data.cwl
```

Instead of:
```
steps:
  - id: 01_validate_data
    in:
      - id: feat_in
        source:
          - feat_in
    out:
      - id: validated_feat
    run: >-
      /home/siamcat_cwl/CWL/tools/01_validate_data.cwl
```

**Comment by mayacoda** [comment]
This is an extremely useful feature, especially for those who want to version control and share their workflows with git.

**Comment by ivanbatic** [comment]
Done with https://github.com/rabix/composer/commit/96a07aa5b79a1ab200af2ce22f035962862c0673
Will be published with the stable 1.0.
